nnpdf_data/nnpdf_data/commondata/ATLAS_SINGLETOP_13TEV/metadata.yaml

```diff
@@ -1,60 +1,61 @@
 setname: ATLAS_SINGLETOP_13TEV
-version: 1
-version_comment: Port of old commondata
+version: 2
+version_comment: Implementation of new data
 nnpdf_metadata:
   nnpdf31_process: SINGLETOP
   experiment: ATLAS
 arXiv:
   url: https://arxiv.org/abs/1609.03920
   journal: JHEP 04 (2017) 086
 iNSPIRE:
-  url: ''
+  url: https://inspirehep.net/literature/1486394
 hepdata:
-  url: ''
-  version: -1
+  url: https://www.hepdata.net/record/ins1486394
+  version: 1
 implemented_observables:
 - observable_name: TCHANNEL-XSEC
   observable:
     description: Heavy Quarks Total Cross Section
     label: ATLAS single top $R_{t}$ 13 TeV
     units: ''
   process_type: INC
-  tables: []
+  tables: [3, 4]
   npoints: []
   ndata: 1
   plotting:
-    kinematics_override: inc_sqrt_scale
+    kinematics_override: identity
     dataset_label: ATLAS single top $R_{t}$ 13 TeV
     plot_x: idat
   kinematic_coverage:
-  - k1
-  - k2
-  - k3
+    - zero
+    - m_t2
```

Comment:
If you want to put `m_t` feel free to make the change (up to you, you are the one that are going to be using this dataset the most I guess).